File: Java/lab10/src/main/java/org/example/Receiver.java
package org.example;

public class Receiver extends WorkerDecorator {
    public Receiver(Worker worker) {
        super(worker);
    }
    public  String makeJob(){
        return " Create order.";
    }
    @Override
    public String doWork() {
        return super.doWork()+makeJob();
    }
}

File: Python/lab10/Receiver.py
from Worker import Worker
from WorkerDecorator import WorkerDecorator

class Receiver(WorkerDecorator):
    def __init__(self, worker: Worker):
        super().__init__(worker)

    def makeJob(self) -> str:
        return " Create order."

    def doWork(self) -> str:
        return super().doWork() + self.makeJob()
